packages/loopback/lib/router/SwaggerRouter.ts

```diff
@@ -192,9 +192,13 @@ class Endpoint {
   }
 }
 
+// NOTE(bajtos) We cannot avoid usage of any here, because we are returning
+// a list of method call arguments, which can contain any types.
 function buildOperationArguments(operationSpec: OperationObject, request: ParsedRequest,
-    pathParams: {[key: string]: any}): any[] {
-  const args: String[] = [];
+  // tslint:disable-next-line:no-any
+  pathParams: {[key: string]: string}): any[] {
+  const args = [];
```

Comment:
This `args` array appears to get an inferred type of `never[]` instead of `any[]`, if I understand the build failure correctly. This has also been opened as #153